lib/terraforming/resource/s3.rb

```diff
@@ -53,7 +53,7 @@ def bucket_policy_of(bucket)
       end
 
       def buckets
-        @client.list_buckets.buckets.select { |bucket| same_region?(bucket) }
+        @client.list_buckets.map(&:buckets).flatten.select { |bucket| same_region?(bucket) }
```

Comment:
__[rubocop]__
* Line is too long. [92/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)